app/controllers/votes_controller.rb

```diff
@@ -3,22 +3,15 @@ class VotesController < ApplicationController
 
   def create
     vote = current_quote.votes.new vote_params.merge(user: current_user)
-    if vote.save
-      flash[:notice] = "Voted!"
-    else
-      flash[:alert] = "Something went wrong!"
-    end
-    redirect_to quotes_path
+    vote.save
+    redirect_to quotes_path, notice: "Voted!" # error thrown if save fails
```

Comment:
`.save` won't actually throw an error if it fails. You're looking for `.save!`. Although I prefer to `if/else` on the `.save` and show validation errors to the user in most cases.